rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -140,3 +147,43 @@ def test_last_queried_ts(tokens, freezer):
         assert len(after_second_query) == 1
         assert after_second_query[0][0] == 'last_queried_timestamp'
         assert int(after_second_query[0][1]) >= continuation
+
+
+def test_cache_is_per_token_type(ethereum_inquirer):
+    """This test makes sure that tokens info cache is per token type"""
+    def query_token_info(token_kind):
+        return _query_or_get_given_token_info(
+            evm_inquirer=ethereum_inquirer,
+            evm_address=ADDR,
```

Comment:
instead of creating a global variable that persists you can make it here

```suggestion
            evm_address=make_evm_address(),
```

Reply:
No, can't be done like that since the address will be regenerated then for every token info request. Bu it has to be the same

Reply:
Than maybe create it inside the test? I don't see why it should be a global variable